django/contrib/gis/db/backends/postgis/schema.py

```diff
@@ -32,15 +33,17 @@ def _create_index_sql(self, model, *, fields=None, **kwargs):
         elif field.dim > 2 and not field.geography:
             # Use "nd" ops which are fast on multidimensional cases
             field_column = "%s %s" % (field_column, self.geom_index_ops_nd)
-        if kwargs.get('name') is None:
-            index_name = '%s_%s_id' % (model._meta.db_table, field.column)
-        else:
-            index_name = kwargs['name']
+
+        name = kwargs.get("name")
+        if not name:
+            name = self._create_index_name(
+                table, [field.column], kwargs.get("suffix", "")
+            )
```

Comment:
We shouldn't change the current logic and use `_id` instead of `suffix` from `kwargs`.

Reply:
1. Current main:
`source"."mymodel_geometry_id`
2. Current PR (Allows user to specify suffix - consistent with posgresql backend):
`mymodel_geometry_26ff3048`
3. Desired change (User cannot specify suffix):
`mymodel_geometry_26ff3048_id`

You want option 3, correct?

Reply:
> Current PR (Allows user to specify suffix - consistent with posgresql backend):

Users cannot specify `prefix` it is an option that Django uses internally. I think we should keep the current prefix `_id` for backward compatibility.